src/assignments/ray/lesson_2_answers.clj

```diff
@@ -1,3 +1,34 @@
 (ns lesson-2-answers)
 
 ;;; Enter your answers in here.
+(def words_1 ["meat" "mat" "team" "mate" "eat"])
```

Comment:
In clojure (and other lisp based languages) we don't use the underscore to
separate tokens, nor do we use CamelCasing.

Thus, you should use  words-1  instead of words_1
and to-set instead of toS.